src/concept/main.tex

```diff
@@ -1,42 +1,42 @@
 % !TeX root = ../../main.tex
 
 \chapter{Konzept}
-In diesem Abschnitt wird kurz unser Konzept für die Umsetzung von Sfm erläutert. 
+In diesem Abschnitt wird kurz unser Konzept für die Umsetzung von SfM erläutert. 
 Insbesondere wird dabei erläutert, was in den einzelnen Schritten durchgeführt wird, und warum diese Schritte notwendig sind.
 Details zur Implementierung werden in Kapitel~\ref{sec:implementation} beschrieben.
-Das Konzept ist eine Kommandozeilen-Applikation, welche eine Pipeline mit Prozessen zur Rekonstruktion von Bildpunkten bereitstellt.
+Konkret soll eine Kommandozeilen-Applikation entwickelt werden, welche eine Pipeline mit Prozessen zur Rekonstruktion von Bildpunkten bereitstellt.
 Die Pipeline ist in Abb.~\ref{fig:concept-pipeline} dargestellt.
 Parameter, die für die einzelnen Prozesse benötigt werden, werden als Kommandozeilen-Argumente übergeben.
 
 Der erste Prozess \emph{Kalibrierung} bestimmt die Parameter der Kamera, mit der die Bilder zur Rekonstruktion aufgenommen wurden.
 Dieser Schritt ist nötig, da bei der Rekonstruktion die Kalibrierungsmatrix K benötigt wird.
 Diese Matrix enthält Informationen über die Brennweite der Kamera und der Position des Hauptpunkts im Bild.
-Des Weiteren wird bei der Kalibrierung auch die Verzerrungsparameter der Kameralinse bestimmt.
+Des Weiteren werden bei der Kalibrierung auch die Verzerrungsparameter der Kameralinse bestimmt.
 Damit kann die Verzerrung in den Bildern reduziert werden, was zu genaueren Ergebnissen beim Rekonstruieren führt.
 Die Kalibrierung wird mit Hilfe eines Kalibrierungsobjekts durchgeführt.
-Es wurde sich bewusst gegen die Auto-Kalibrierung entschieden, da dies nicht immer möglich ist sie anzuwenden~\cite{remondino_2005}.
+Es wurde sich bewusst gegen die Auto-Kalibrierung entschieden, da es nicht immer möglich ist diese anzuwenden~\cite{remondino_2005}.
 Dementsprechend müssen zusätzliche Bilder für die Kalibrierung vom Nutzer bereitgestellt werden.
 Der Nutzer muss dabei nur einen Ordnerpfad angeben, in dem sich die Bilder für die Kalibrierung befinden.
 Das Kalibrieren der Kamera kann sehr zeitaufwendig sein.
-Daher wird die Möglichkeit bereitgestellt, die Kameraparameter in einer Datei zu exportieren.
-Diese Datei kann dann geladen werden, so dass eine weitere Kalibrierung mit dem Schachbrettmuster nicht mehr nötig ist. 
+Daher wird die Möglichkeit bereitgestellt, die Kameraparameter in eine Datei zu exportieren.
+Diese Datei kann dann geladen werden, so dass eine erneute Kalibrierung mit dem Schachbrettmuster nicht mehr nötig ist. 
```

Comment:
Das Schachbrettmuster sollten wir entweder hier schon vorher explizit erwähnen oder in der Implementierung